packages/taquito/src/contract/rpc-contract-provider.ts

```diff
@@ -759,6 +763,87 @@ export class RpcContractProvider extends Provider implements ContractProvider, S
 
     return batch;
   }
+
+  /**
+   *
+   * @description Stake the given amount for the source. The source must be a delegator to be allowed to stake
```

Comment:
might want to mention `before adaptive_issuance_launch_cycle the source must be a delegator to be allowed to stake, after the head block's cycle reached adaptive_issuance_launch_cycle any implicit can stake`

Reply:
bump